api/app/services/shadow_mode_processor.py

```
@@ -0,0 +1,483 @@
+            context_messages: Previous messages for conversation context (optional)
+
+        Returns:
+            ShadowResponse with answer and confidence, or None on error
+        """
+        try:
+            # Scrub PII from question
+            sanitized_question = self._scrub_pii(question)
+
+            # Detect version from question text using shared VersionDetector
+            # This provides consistent detection with the RAG pipeline
+            detected_version, version_confidence, clarifying_question = (
```

Ruff (0.14.10)

Unpacked variable `clarifying_question` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)